frontend/src/pages/videoPlayer.tsx

```diff
@@ -36,18 +37,11 @@ const VideoPlayer: React.FC = () => {
   }
 
   return (
-    <div className="flex flex-col min-h-screen bg-white">
-      <header className="sticky top-0 z-10 bg-white border-b">
-        <div className="container flex items-center justify-between h-14 px-4">
-          <Link to="/" className="text-2xl font-bold text-blue-600">
-            Pro Tube
-          </Link>
-        </div>
-      </header>
-
+    <div className="flex flex-col min-h-screen bg-black text-white">
+      <br />
       <main className="flex-1 container px-4 py-6">
         <div className="w-full max-w-lg mx-auto space-y-4">
-          <div className="aspect-w-16 aspect-h-9">
+          <div className="w-full h-screen">
```

Comment:
Try to put changes affecting different parts of the project in separate PRs. In this case, don't put in a PR for the backend changes in the layuout of the frontend